content/en/docs/rate-limits.md

```diff
@@ -3,131 +3,148 @@ title: Rate Limits
+are integrating Let's Encrypt into your operations, please review our [Integration Guide](/docs/integration-guide)
+to ensure smooth operation. In particular, we recommend using one ACME account for many
+customers. If you are developing an ACME client, debugging the usage of one, or configuring
+an automated system, please take advantage of our [staging environment](/docs/staging-environment),
+which is governed by higher rate limits and issues false certificates.
+
+We have no way of temporarily overriding the rate limits. For very large providers who are
+encountering rate limits due to sheer volume, we provide a [rate limiting form](https://goo.gl/forms/plqRgFVnZbdGhE9n1)
+that can be used to request higher rate limits. Since these requests can take several weeks
+to process, the form is not suitable for addressing temporary situations that will
+automatically be resolved when any encountered rate limits no longer apply.
+
+Revoked certificates still count against our rate limits and remain in the public record.
+All issued certificates can be found using [crt.sh](https://crt.sh/), which uses public
+[Certificate Transparency](https://www.certificate-transparency.org) logs. In general,
+certificate revocation should only be considered under very particular circumstances, such as the
+compromise of a certificate's private key. Securely deleting a certificate's private key
+is usually sufficient to decommission the certificate due to perfect forward secrecy.
+Revocation of a certificate only invalidates its trust for future use.
+
+# Rate Limits
+
+Message: too many certificates already issued for exact set of domains
+
+Limit: 5 <a href="#duplicate-certificate">duplicate certificates</a> / 7 days
+
+Hitting this limit is often the result of using the following practices that should be avoided:
+* Deleting valid certificates (and their private keys) when attempting to resolve webserver configuration problems
+* Spinning-up ephemeral instances that acquire new certificates from the CA rather than utilizing an existing certificate
+
+***
+
+Message: too many failed authorizations recently
+
+Limit: 5 failed validation attempts / <a href="#FQDN">FQDN</a> / ACME account / hour
+
+Hitting this limit is often the result of:
+* Improper webserver or DNS configuration
+* Misunderstanding of an [ACME challenge](/docs/challenge-types)
+
+***
+
+Message: too many certificates already issued
+
+Limit: 50 certificates / <a href="#apex-domain-name">apex domain name</a> / week
+
+<a href="#duplicate-certificate">Duplicate certificates</a> do **not** count against this limit.
+
+***
+
+Limit: 100 <a href="#SAN">SANs</a> / certificate
+
+***
+
+Message: too many new orders recently
+
+Limit: 300 new orders / ACME account / 3 hours
+
+If a new order is requested and there already exists a pending order covering the exact same
+<a href="#FQDN">FQDNs</a>, the pending order will be returned and a new order will **not**
+be created.
+
```

Comment:
How about something like this?

> This limit may be reached due to a client software bug or due to overly-aggressive efforts to parallelize a high volume of certificate issuance. Most users should simply decrease the parallelism of their certificate requests, or arrange for certificate requests by separate parts of their organizations to use separate ACME accounts. If you are a large hosting provider and reach this limit in the course of legitimate and otherwise successful certificate issuance on behalf of your customers, you may contact Let's Encrypt to discuss your situation.

Reply:
I never really considered that aspect. I've personally encountered this a lot with simple ACME client scripts that submit a new order matching the pending order upon a nonce timeout.